### ISSUE TITLE
1.16.5 - Datapack doesn't do anything

### ISSUE THREAD

**Issue by Roadhog360** [EdwinMindcraft/origins-architectury#225]
Sorry if 1.16.5 isn't supported any more but as no mention of it is made, here I go

I was trying to make the Enderian not able to eat Farmer's delight pumpkins and noticed there was no tags, so I had to go make a datapack since I can't change the pumpkin pies to be tagged with Craft Tweaker.

So I made this datapack to add a tag, and added the pumpkin foods in craft tweaker and I noticed it didn't work, so I tried removing a handful of Enderian abilities to be sure I did it right, nothing changed, all abilities are there, yet the datapack still loads fine and shows in the list.

So maybe I'll go on to the next change I wanted to make, changing the one mining multiplier in Feline, to allow stone to be mined slowly. So I changed the number from -1 to 0.2. Nothing changed in-game. So I changed it to 1. Nothing, again.

Then after all that I tried adding `"replace": true,` to all of the files. STILL nothing happens.

[datapack.zip](https://github.com/EdwinMindcraft/origins-architectury/files/9277749/datapack.zip)

So I've basically tried everything, made sure there were no errors in the console and the datapack is really loaded. What am I doing wrong?

To give a recap, this datapack, as a test, is supposed to remove the feline mining debuff by changing the -1 to 1 (specifically this way so I can test if this is the right number, then change it to 0.2 later) and then also disable most of the Enderian powers as a test. Then I add a pumpkin_foods item tag but only actually added pumpkin_pie to it because I didn't want to have modded items there causing errors if I decided to make this a full-on datapack I share online, so I instead added the pumpkin foods with CraftTweaker.

**Comment by Roadhog360** [comment]
So, I've done a bit more digging, and as it turns out, modifying the same files directly in the jar works, so data packs are definitely bugged. It seems jar modifications are the only way to actually change any default Origin properties.

**Comment by EdwinMindcraft** [comment]
Use `loading_priority` in your origins and powers.

**Comment by Roadhog360** [comment]
How do you do that?

**Comment by EdwinMindcraft** [comment]
See [Origin JSON](https://origins.readthedocs.io/en/latest/json/origin/) and [Power JSON](https://origins.readthedocs.io/en/latest/json/power/) on the origins documentation.

**Comment by Roadhog360** [comment]
Sorry for the delay. That mostly fixed it, all of my modifications work except one.
[rebalance.zip](https://github.com/EdwinMindcraft/origins-architectury/files/9386513/rebalance.zip)
The Enderian is completely ignoring the changes I made to its pumpkin_hate power. I tried to make it use a tag, but no matter what, even if I set the loading priority to 100000000 or remove everything from the pumpkin_foods tag I created, the changes are simply ignored and it only blocks the use of the default pumpkin pie even if I remove that from the tag.
What am I doing wrong? This exact configuration works fine when I put it in the jar. There are no errors in console.
